GVRf/Framework/framework/src/main/java/org/gearvrf/GVRComponent.java

```diff
@@ -144,7 +143,15 @@ public void setEnable(boolean flag) {
             onDisable();
         }
     }
-    
+
+    /**
```

Comment:
There is no GVRComponentGroup. should this method stay?

Reply:
no, i will remove it